pkg/diagnostics/cluster/aggregated_logging/daemonsets.go

```diff
@@ -0,0 +1,109 @@
+package aggregated_logging
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapisext "k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+const daemonSetNoLabeledNodes = `
+There are no nodes that match the selector for DaemonSet '%[1]s'.  
+An example of a command to target a specific node for this DaemonSet:
+
+   oc label node/ip-172-18-2-170.ec2.internal %[2]s
```

Comment:
Which indentation are you refering to?  The initial statement like "There are..." or the command example: "  oc label"?   Do you maybe have an example to reference I can follow?